app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# Create Confirmations for Users
+class ConfirmationsController < Devise::ConfirmationsController
+  respond_to :json
+
+  def new
+    super
+  end
+
+  def create
+    super
+    @front_end_path = concatenate_path
```

Comment:
This should be added to [`DeviseCustomMailer#confirmation_instructions`](https://github.com/pieforproviders/pieforproviders/pull/208/files#diff-62839391a18852f6d923c89329ad71ffR8).